client/src/test/java/com/sinch/sdk/models/E164PhoneNumberTest.java

```diff
@@ -35,7 +35,9 @@ void stringValue() {
 
   @Test
   void valueOf() {
-    assertEquals(E164PhoneNumber.valueOf(phoneNumber), E164PhoneNumber.valueOf(phoneNumber));
+    assertEquals(
+        E164PhoneNumber.valueOf(phoneNumber).stringValue(),
+        E164PhoneNumber.valueOf(phoneNumber).stringValue());
```

Comment:
Still, I don't understand this test: both terms are the same

Reply:
Agree; sorry to waste your time
Fixed by removing the test (the `valueOf` is indirectly tested by previous test related to `stringValue`)